packages/meditrak-server/src/routes/optionSets/EditOptionSets.js

```diff
@@ -15,6 +15,10 @@ export class EditOptionSets extends EditHandler {
   }
 
   async editRecord() {
-    await this.updateRecord();
+    const optionSet = await this.models.optionSet.findById(this.recordId);
+    const originalData = await optionSet.getData();
+    const updatedModel = { ...originalData, ...this.updatedFields };
+
+    return this.models.optionSet.updateById(this.recordId, updatedModel);
```

Comment:
What's the benefit of this specific handling?

Reply:
I thought it was odd, but it was one of the few endpoints with a specific handler in the dataAccessors folders so I thought there must have been a reason.

If you also don't understand/remember why it was done this way then maybe I'll just get rid of it.

Reply:
The original commit was in [this PR](https://github.com/beyondessential/meditrak-server/pull/249/files), where there was an obvious reason for the `editOption` to be handled specifically, but less obvious for `editOptionSet`. I'd be keen to drop it and then highlight it as an area for retesting.